src/connectors/fritz.ts

```diff
@@ -1,9 +1,28 @@
 export {};
 
-Connector.playerSelector = 'div.layoutnowonair_reload_active';
+Connector.playerSelector = '#main';
 
-Connector.artistSelector = 'div.teasertext > h3 > a > span.manualteasertitle';
+Connector.artistSelector = '.artist';
 
-Connector.trackSelector = 'div.teasertext > h3 > h3 > span.manualteasertitle';
+Connector.trackSelector = '.songtitle';
 
-Connector.isPlaying = () => Util.hasElementClass('.player', 'player_playing');
+Connector.pauseButtonSelector = '.audioplayer-playing';
+
+const filter = MetadataFilter.createFilter({
+	artist: (artist) => {
+		// If the artist string contains a semicolon, split the string by the semicolon
+		// and return the first element (the part of the string before the semicolon).
+
+		// Replace .feat with a semicolon
+		artist = artist.replace('.feat', ';');
```

Comment:
Looks like you need to fix this by perhaps using another variable within the function. 

e.g. `const updatedArtist = artist.replace('.feat', ';');` and using this within the function?

Reply:
Yeah tbj i just ignored the eslint error. But i fixed it accordingly.